src/Miunie.WindowsApp/ViewModels/SettingsPageViewModel.cs

```diff
@@ -32,7 +32,7 @@ public SettingsPageViewModel(MiunieBot miunie, ILogReader logReader, TokenManage
 
         public string BotAvatar => _miunie.MiunieDiscord.GetBotAvatarUrl() ?? DefaultAvatarUrl;
 
-        public IEnumerable<object> Logs => _logReader.RetrieveLogs(10).Select(m => new { Message = m });
+        public string Logs => string.Join(Environment.NewLine, _logReader.RetrieveLogs(10).Select(m => $"{m}"));
```

Comment:
I thought `RetrieveLogs` returns an `IEnumerable<string>`. In that case why do we have that `Select` statement there when the result of that is also an `IEnumerable<string>`? 🤔 

Is there something I'm missing?

Reply:
Nope, you are not missing anything there. It was leftover code from a refactor where I was hoping to get a DateTime stamp but only IEnum of string. Removed!